org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/XMLServerLauncher.java

```diff
@@ -27,6 +29,16 @@
 	 * and output streams.
 	 */
 	public static void main(String[] args) {
+		final String username = System.getProperty("http.proxyUser", "");
+		final String password = System.getProperty("http.proxyPassword", "");
+		Authenticator.setDefault(new Authenticator() {

```

Comment:
Test configuration of Authentificator should be done only if username + password is set, no?

Why not using lambda instead of new Athenticator?

Reply:
> Test configuration of Authentificator should be done only if username + password is set, no?

Fixed.

> Why not using lambda instead of new Athenticator?

Authenticator is the abstract class.